operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -194,15 +194,12 @@ type CentralDBSpec struct {
 }
 
 // CentralDBEnabled is a type for values of spec.central.db.enabled.
-//+kubebuilder:validation:Enum=Default;Enabled
+//+kubebuilder:validation:Enum=Default
```

Comment:
Shrinking the allowed set of values is not a compatible change.
TBH neither is growing the set, but removing a previously allowed value is more likely to cause problems.

```suggestion
//+kubebuilder:validation:Enum=Default
```

Reply:
You are right, it's much safer